### PR TITLE
Add missing testinfra tests

### PR DESCRIPTION
**Pull request by lindenb1**

### CODE REVIEW

**Comment by lindenb1** [comment]
sysdig is currently not working, for some reason even when installing DKMS and building the kernel module for sysdig, the system isn't able to find the kernel module after installation.

File: .zuul.yaml
Comment (gndrmnn) [inline .zuul.yaml:1]: Use new declaration like in collection-services, where files are declared in jobs

File: README.md [hunk 1]
```diff
@@ -4,45 +4,44 @@ Documentation: https://osism.github.io/docs/guides/configuration-guides/commons/
...
+| sshconfig      | [sshconfig.py](molecule/delegated/tests/sshconfig.py)                |
+| state          | [state.py](molecule/delegated/tests/state.py)                        |
+| sysctl         | [sysctl.py](molecule/delegated/tests/sysctl.py)                      |
+| systohc        | [systohc.py](molecule/delegated/tests/systohc.py)                    |
+| timezone       | [timezone.py](molecule/delegated/tests/timezone.py)                  |
+| trivy          | [trivy](molecule/delegated/tests/trivy)                              |
+| vault_import   |                                                                      |
+| vault_init     |                                                                      |
+| vault_seal     | [prepare/vault_seal.yml](molecule/delegated/prepare/vault_seal.yml)  |
+| vault_unseal   | [prepare/vault_seal.yml](molecule/delegated/prepare/vault_unseal.yml)|
```
Comment (gndrmnn) [inline README.md:47]: Add 2 links, and fix the unseal link

File: .zuul.yaml
Comment (gndrmnn) [reply .zuul.yaml:1]: Also: We need one run with all the tests

File: molecule/delegated/prepare/vault_init.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+#  Preparing the vault and executing the role is the test
+
+- name: Include Docker and HashiCorp Vault deployment
+  ansible.builtin.include: vault_setup.yml
+
+# - name: Make sure that required module is installed
+#  ansible.builtin.pip:
+#    name: ansible-modules-hashivault
+# -> Hotfixed, now using: https://github.com/TerryHowe/ansible-modules-hashivault.git (branch: ansible-collection)
```
Comment (gndrmnn) [inline molecule/delegated/prepare/vault_init.yml:11]: Can be removed

File: roles/vault_import/tasks/main.yml [hunk 3]
```diff
@@ -5,8 +5,8 @@
     file: "{{ vault_secrets_path }}"
 
 - name: Create secret
-  no_log: true
-  hashivault_secret:
+  no_log: false
```
Comment (gndrmnn) [inline roles/vault_import/tasks/main.yml:8]: ``true``

**Review by gndrmnn — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (lindenb1) [reply README.md]: DONE

File: .zuul.yaml
Comment (lindenb1) [reply .zuul.yaml:1]: DONE

File: molecule/delegated/prepare/vault_init.yml (hunk 2 above)
Comment (lindenb1) [reply molecule/delegated/prepare/vault_init.yml]: DONE

File: roles/vault_import/tasks/main.yml (hunk 3 above)
Comment (lindenb1) [reply roles/vault_import/tasks/main.yml]: DONE

**Pull request merged**